packages/functional-tests/playwright.config.ts

```diff
@@ -49,19 +49,6 @@ const config: PlaywrightTestConfig<TestOptions, WorkerOptions> = {
           },
         } as Project)
     ),
-    {
-      name: 'stub',
```

Comment:
I think you still need to delete this from the package.json since this project is now gone

https://github.com/mozilla/fxa/blob/3e47a1dda7a5e00198506ce0e7aacddb52b66a59/packages/functional-tests/package.json#L7